supervisor_types.go

```diff
@@ -1,6 +1,8 @@
 package druid
 
-import "time"
+import (
+	"time"
+)
```

Comment:
nit: This is probably just an unnecassary lefover cause adding and rmoving imports during development.

Reply:
Iit was actually used in one of the definitions, now changed to string for consistency. Tx!